src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -155,6 +124,70 @@ const SpendingByCFDAContainer = ({ onRedirectModalClick, activeTab }) => {
         setOrder(direction);
     };
     const defCodes = useSelector((state) => state.covid19.defCodes);
+    const dispatch = useDispatch();
+
+    const updateAdvancedSearchFilters = (e) => {
+        e.preventDefault();
+        dispatch(clearAllFilters());
+        dispatch(resetAppliedFilters());
+        const cfdaData = results.find((cfda) => cfda.code === e.target.value);
+        dispatch(applyStagedFilters(
+            Object.assign(
+                {}, defaultAdvancedSearchFilters,
+                {
+                    defCodes: new CheckboxTreeSelections({
+                        require: defCodes.map((code) => code.code),
+                        exclude: [],
+                        counts: [{ value: "COVID-19", count: defCodes.length, label: "COVID-19 Response" }]
+                    })
+                },
+                {
+                    selectedCFDA: OrderedMap({
+                        [cfdaData.code]: {
+                            program_number: cfdaData.code,
+                            program_title: cfdaData.description,
+                            popular_name: cfdaData.description,
+                            identifier: cfdaData.code
+                        }
+                    })
+                }
+            )
+        ));
+        Router.history.push('/search');
+    };
+
+    const parseRows = () => (
+        results.map((row) => {
+            const rowData = Object.create(BaseSpendingByCfdaRow);
+            rowData.populate(row);
+            let link = rowData.name;
+            if (rowData._code) {
+                link = (
+                    <button
+                        className="assistance-listing__button"
+                        value={rowData._code}
+                        onClick={updateAdvancedSearchFilters}>
+                        {rowData.name}
+                    </button>
+                );
+            }
+            if (activeTab === 'loans') {
+                return [
+                    link,
+                    rowData.faceValueOfLoan,
```

Comment:
This is my bad - I realize the order of the column headings was updated to match the latest mock but not the order of the results. It should be `obligation`, `outlay`, `faceValueOfLoan`, `count`